### DISCUSSION TITLE
My First Extension!!! Keep you happy while you develop

### DISCUSSION THREAD

**Discussion by hdulay** [microsoft/vscode-discussions#1278] [Extension Show and Tell]
The docs and examples are great. Thanks for the hard work around VSCode extension development.

https://marketplace.visualstudio.com/items?itemName=1Schema.obsequious

**Comment by hegdeashwin** [comment]
Nice to see this.

@hdulay Did you faced this issue before https://github.com/microsoft/vscode-discussions/discussions/1281

**Reply by hdulay** [reply]
I have not. Which vscode version are you running?

**Comment by isidorn** [comment]
Glad to hear you liked the extension development flow!

Have you considered making your extension a Language Model extension https://code.visualstudio.com/api/extension-guides/language-model
